### PR TITLE
Issue 184 

### PR DESCRIPTION
**Pull request by wangsnuxiao**
Explore and Days favorite button

### CODE REVIEW

**Comment by sballan** [comment]
Great stuff!

I was just starting my review, and noticed that on your branch the title of a day and the venues do not appear:

Screen Shot 2021-11-30 at 4 30 01 PM
 
I figured I'd leave this comment right away before continuing with my review

**Comment by sballan** [comment]
> Great stuff!
> 
> I was just starting my review, and noticed that on your branch the title of a day and the venues do not appear:
> 
> Screen Shot 2021-11-30 at 4 30 01 PM
> 
> I figured I'd leave this comment right away before continuing with my review

As we discussed, this has been resolved

**Comment by yp1383** [comment]
Oh great! Accidently remove that stuff. Thx

File: explore/templates/explore/_exlpore_day_venuelist.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                                <button class="button">
+                                    <a href="/resources/venues/for_yelp_id/{{ dv.dayvenue.venue.yelp_id }}">
+                                        View on Yelp
+                                    </a>
+                                </button>
+                            </footer>
+                        </div>
+                    </div>
+                    <div class="columns">
+                        <button id="btn_{{dv.dayvenue.id}}" class="button is-outline for-collector column" data-dv_id="{{dv.dayvenue.id}}">More Details...</button>
```
Comment (sballan) [inline explore/templates/explore/_exlpore_day_venuelist.html:56]: What is the class `for-collector`?  I see it in many places, I'm not sure what it does

File: explore/views.py [hunk 2]
```diff
@@ -3,15 +3,36 @@
...
 
 
 def explore(requets):
     context = {}
+    if "Error_Message" in requets.session:
+        context["error"] = requets.session["Error_Message"]
+        del requets.session["Error_Message"]
+    elif "success_Message" in requets.session:
+        context["message"] = requets.session["success_Message"]
+        del requets.session["success_Message"]
```
Comment (sballan) [inline explore/views.py:21]: For consistency, this should probably be "Success_Message", not "success_Message"

File: explore/views.py [hunk 3]
```diff
@@ -3,15 +3,36 @@
 
 # Create your views here.
 from dayplanner.services import yelp_client
-from resources.days.models import Day
+from resources.days.models import Day, FavoriteDay, DayVenue
 from resources.categories.models import Category
+from resources.venues.models import FavoriteVenue
+
+
+ERROR_FAV_NoLOGIN = "To Save your Favourite day, Please Log in First"
```
Comment (sballan) [inline explore/views.py:11]: Name this `ERROR_FAV_NO_LOGIN`

File: explore/templates/explore/_exlpore_day_venuelist.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
+{% for dv in dayvenue_list %}
```
Comment (sballan) [inline explore/templates/explore/_exlpore_day_venuelist.html:1]: Typo in the name of this file.  It should be `_explore_day_venuelist.html`

File: explore/templates/explore/_exlpore_day_venuelist.html (hunk 1 above)
Comment (yp1383) [reply explore/templates/explore/_exlpore_day_venuelist.html:56]: I have no idea. I just copy and paste from other place

**Review by sballan — APPROVED** [review]
Looks great!

**Pull request merged**